lib/builder/next_builder_library.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:analyzer/dart/element/element.dart';
+
+class NextBuilderLibrary {
+  NextBuilderLibrary(this.element);
+  final ClassElement element;
+
+  String build() {
+    final ClassElement elementSuperType =
```

Comment:
Here we have to introspect the second type argument for the Module class. 

(i.e **NuRoute<NuModule, FriendRequestArgs, void>**)

After that, we can access the field members and all the properties of that ClassElement.